securedrop/models.py

```diff
@@ -35,6 +35,10 @@
 ARGON2_PARAMS = dict(memory_cost=2**16, rounds=4, parallelism=2)
 
 
+def generate_otp_secret() -> bytes:
```

Comment:
Type checking is failing, as `bytes` are specified but a `str` is expected in the `models.py` call.

Reply:
It should be fixed now